### PR TITLE
Remove breadcrumbs code from AMO

### LINKED ISSUE #4250: Remove breadcrumbs code from addons website
Many pages of addons website has breadcrumbs, which is currently hidden [in restyle.less](https://github.com/mozilla/addons-server/blob/53c01f6dad05027811e823b056962ffe6cf148aa/static/css/restyle/restyle.less#L1195-L1197) . If the breadcrumbs displays with current UI of the website, it looks bad & weird. Also @jvillalobos [mentioned here](https://github.com/mozilla/addons-server/issues/4173#issuecomment-270005364) it's okay to remove them entirely as they are hidden for a long long time.

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
Fixes #4250

### CODE REVIEW

File: src/olympia/editors/helpers.py [hunk 1]
```diff
@@ -79,67 +79,6 @@ def editor_page_title(context, title=None, addon=None):
 
 @register.function
 @jinja2.contextfunction
-def editors_breadcrumbs(context, queue=None, addon=None, items=None,
```
Comment (tofumatt) [inline src/olympia/editors/helpers.py]: Didn't you log in to the admin tools/editor tools locally and test these were hidden? They seem useful if not hidden (and restyle might not have touched the editor tools as much).

If they were hidden or @jvillalobos says these aren't needed it's fine.

Comment (tofumatt) [inline src/olympia/editors/helpers.py]: Never mind, just saw they are hidden everywhere. All good!

**Comment by tofumatt** [comment]
The tests are passing now, thanks. Did you have the chance to test this locally?

**Comment by tofumatt** [comment]
Also so much red! red-celebrate

**Comment by atiqueahmedziad** [comment]
@tofumatt yap, tested it locally. Absolutely no sign of breadcrumbs :)

**Comment by muffinresearch** [comment]
As a heads-up there was a regression https://github.com/mozilla/addons-server/issues/4265 due to a opening block being accidentally removed - it was a simple fix - just letting you know to watch out for similar things in future.

Looking over the patch again this part looks like it might be going too far too:  https://github.com/mozilla/addons-server/pull/4256/files#diff-6c32629a02f093e5cbfe1d1813370181L19 since it's removing the if entirely which now means there's not the title if sort is true. I think that should be checked it might require re-instating the `if` statement and just removing the breadcrumbs.

**Comment by muffinresearch** [comment]
> Looking over the patch again this part looks like it might be going too far too: https://github.com/mozilla/addons-server/pull/4256/files#diff-6c32629a02f093e5cbfe1d1813370181L19 since it's removing the if entirely which now means there's not the title if sort is true. I think that should be checked it might require re-instating the if statement and just removing the breadcrumbs.

Fixed in https://github.com/mozilla/addons-server/commit/31d59da55bf1f5512f7eebe24fc3be634a087b35 Thanks @atiqueahmedziad

**Pull request merged**